.buildkite/scripts/bootstrap.sh

```diff
@@ -34,11 +39,11 @@ if ! (yarn kbn bootstrap "${BOOTSTRAP_PARAMS[@]}" || yarn kbn bootstrap "${BOOTS
   # So, we should just delete node_modules in between attempts
   rm -rf node_modules
 
+  export MOON_LOG=debug
```

Comment:
This provided useful when Moon didn't manage to run some post-install tasks. But we can remove it before merging